onnxruntime/test/util/compare_ortvalue.cc

```diff
@@ -106,17 +135,43 @@ std::pair<COMPARE_RESULT, std::string> CompareFloatResult(const Tensor& outvalue
 }
 
 template <typename T>
-std::pair<COMPARE_RESULT, std::string> IsResultExactlyMatch(const Tensor& outvalue, const Tensor& expected_value) {
+std::pair<COMPARE_RESULT, std::string> IsResultExactlyMatch(const Tensor& outvalue, const Tensor& expected_value,
+    bool dropout = false) {
```

Comment:
Same here. 
You can introduce another utility function to handle the special case if need.